CMakeLists.txt

```diff
@@ -38,11 +38,12 @@ list (APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 # provide user options
 option (BUILD_SHARED_LIBS "Build NLopt as a shared library" ON)
 option (WITH_CXX          "Enable C++-based routines (StoGO)" OFF)
-option (USE_SWIG          "Use SWIG to generate bindings" ON)
+option (WITH_SWIG          "Use SWIG to generate bindings" ON)
 option (BUILD_PYTHON      "Build Python bindings (using SWIG)" ON)
 option (BUILD_GUILE       "Build Guile bindings (using SWIG)" ON)
 option (BUILD_OCTAVE      "Build Octave bindings" ON)
 option (BUILD_MATLAB      "Build MATLAB bindings" ON)
+option (BUILD_TESTS       "Build Tests" ON)
```

Comment:
this is useless, as by default the test executable is not built (excluded from all target)